### DISCUSSION TITLE
Docker Storage equivalent for Orioin

### DISCUSSION THREAD

**Discussion by BitTheByte** [PrefectHQ/prefect#5537] [Q&A] [answered]
Previously we had docker storage that allowed building and pushing docker images. do we still have this feature for Orion? if so, is there any examples? if not, will this ever get implemented again or do we need to manage our custom images?

**Comment by anna-geller** [comment, marked as answer]
It's on our radar and we are aware that many users would like to see similar functionality in Prefect 2.0. I can't make any promises as to how this will look like and when you can expect to see this implemented but we are aware of the problem. The easiest way to stay in touch on that would be to subscribe to the [release-notes](https://discourse.prefect.io/tag/release-notes) tag on Discourse and you will get notified via email about all new releases.
